packages/storybook/src/stories/docs/framework-integration/vue-instructions.stories.mdx

```diff
@@ -3,20 +3,77 @@ import { Meta } from "@storybook/addon-docs";
 <Meta title="Docs/Framework Integration/Vue" />
 
 # Integrate with Vue.js (as Web Components)
-
-> We set up a demo project with Vue.js to show you how to integrate the components into a newly generated Vue.js
+> We set up a demo project with Vue.js (v3) to show you how to integrate the components into a newly generated Vue.js
 > project. Scroll down to see the Stackblitz.
 >
 > These instructions are based on `vue@2.6.11` and [`create-react-app@4.2.3`](https://cli.vuejs.org/).
```

Comment:
We use a newer vue version, don't we?

Reply:
I removed the specific lines which contains version numbers.